File: Basic Question Bank/a015/a015.py
from sys import stdin

for s in stdin:
    row, col = map(int, s.split())
    matrix = []
    for i in range(row):
        matrix.append(stdin.readline().strip().split())
    for i in range(col):
        for j in range(row):
            print(matrix[j][i], end=" ")
        print()

File: Basic Question Bank/a015/a015.java
import java.util.Scanner;

public class a015 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while(sc.hasNext()) {
            int row = sc.nextInt(), col = sc.nextInt();
            int[][] matrix = new int[row][col];
            for(int i = 0; i < row; i++) {
                for(int j = 0; j < col; j++) {
                    matrix[i][j] = sc.nextInt();
                }
            }

            for(int i = 0; i < col; i++) {
                for(int j = 0; j < row; j++) {
                    System.out.print(matrix[j][i] + " ");
                }
                System.out.println();
            }
        }
        sc.close();
    }
}
